pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -38,15 +38,22 @@ var (
 	InternalLBAnnotations = map[configv1.PlatformType]map[string]string{
```

Comment:
These annotations are only for setting up internal LBs — to apply annotations to all LB service resources, gotta add them [here](https://github.com/openshift/cluster-ingress-operator/blob/master/pkg/operator/controller/ingress/load_balancer_service.go#L117)